src/mapbox/mapbox.js

```diff
@@ -66,6 +67,7 @@ const defaultProps = {
   onLoad: noop,
   onError: noop,
   reuseMaps: false,
+  transformRequest: undefined,
```

Comment:
Use `null` instead of `undefined`? I find explicit usage of `undefined` to be tricky as it so easily can be the result of an unintentional coding error.

Reply:
Good catch.  Changed default to `null` and now I only pass it to the constructor if present.  Thanks for the review!